setup.py

```diff
@@ -90,7 +90,7 @@ def read_reqs(req_file):
     description=("Elastic Search Connectors."),
     long_description=long_description,
     author="Search Extract and Transform Team",
-    author_email="tarek@ziade.org",
+    author_email="enterprise-search-ingestion-team@elastic.co",
```

Comment:
should this be `extract-and-transform-team`? I'm not sure if it exists though

Reply:
Yes, this should be renamed. I'll follow-up with @DianaJourdan after her PTO. I double-checked and `enterprise-search-ingestion-team@elastic.co` seems to be the correct address for now. I'll change it as soon as we've a new one.